code/game/objects/structures/crates_lockers/crates/wooden.dm

```diff
@@ -0,0 +1,17 @@
+/obj/structure/closet/crate/wooden
+	name = "wooden crate"
+	desc = "Works just as well as a metal one."
+	material_drop = /obj/item/stack/sheet/mineral/wood
+	material_drop_amount = 6
+	icon_state = "wooden"
+
+/obj/structure/closet/crate/wooden/toy
+	desc = "It has the words \"Clown + Mime\" written underneath of it with marker."
```

Comment:
it is supposed to be a signature, not alluding to heretic, filthy, disgusting clown/mime relationships!